src/lib/ui/layout/DetailRow.tsx

```diff
@@ -2,15 +2,22 @@ import type { PropsWithChildren } from 'react'
 
 interface DetailRowProps {
   label: string
+  sublabel?: string
 }
 
 export function DetailRow({
   label,
+  sublabel,
   children,
 }: PropsWithChildren<DetailRowProps>): JSX.Element {
   return (
     <div className='seam-detail-row'>
-      <p className='seam-row-label'>{label}</p>
+      <div className='seam-detail-row-label-wrap'>
+        <p className='seam-row-label'>{label}</p>
+        {sublabel !== undefined && (
```

Comment:
```suggestion
        {sublabel != null  && (
```

Reply:
This is what I mean, yeah. If we use `null` here, the element will still render in the DOM

Reply:
(Same with the recent change to `DetailSection` with the optional `label` prop)

Reply:
Should be treated the same via `==`

<img width="227" alt="image" src="https://github.com/seamapi/react/assets/721372/e8bc51e6-938f-425f-9c62-471ff2220078">

Reply:
See this:

https://github.com/seamapi/react/assets/87919558/f6f79acc-1620-405a-b29c-b63262a1cd2a

Reply:
Sorry it's because I have `!==` not `!=`—glossed over that, my bad